ale/transformation.py

```diff
@@ -53,12 +53,13 @@ def create_rotations(rotation_table):
                            rotation_table['J2000Ang3'][:-1]]).T
         angles = polyval(scaled_times, coeffs).T
         # ISIS is hard coded to ZXZ (313) Euler angle axis order.
+        # SPICE also interprets Euler angle rotations as negative rotations,
+        # so negate them before passing to scipy.
         time_dep_rot = TimeDependentRotation.from_euler('zxz',
-                                                        angles,
+                                                        -angles,
```

Comment:
Any leads on why this is necessary?

Reply:
Under the particulars section of [eul2m](https://naif.jpl.nasa.gov/pub/naif/toolkit_docs/FORTRAN/spicelib/eul2m.html)

```
      A word about notation:  the symbol

        [ x ]
             i

     indicates a rotation of x radians about the ith coordinate axis.
     To be specific, the symbol

        [ x ]
             1

     indicates a coordinate system rotation of x radians about the
     first, or x-, axis; the corresponding matrix is

        +-                    -+
        |  1      0       0    |
        |                      |
        |  0    cos(x)  sin(x) |.
        |                      |
        |  0   -sin(x)  cos(x) |
        +-                    -+

     Remember, this is a COORDINATE SYSTEM rotation by x radians; this
     matrix, when applied to a vector, rotates the vector by -x
     radians, not x radians.  Applying the matrix to a vector yields
     the vector's representation relative to the rotated coordinate
     system.
```

This is what ISIS uses to actually apply the rotations, so we need to mirror eul2m's behavior of rotating about the negative axis (which is equivalent to rotating by the negative angle).